diff --git a/package/x11r7/xwayland/xwayland.mk b/package/x11r7/xwayland/xwayland.mk
index c66dd471ac6..8128e02d024 100644
--- a/package/x11r7/xwayland/xwayland.mk
+++ b/package/x11r7/xwayland/xwayland.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-XWAYLAND_VERSION = 23.2.4
+XWAYLAND_VERSION = 24.1.6
 XWAYLAND_SOURCE = xwayland-$(XWAYLAND_VERSION).tar.xz
 XWAYLAND_SITE = https://xorg.freedesktop.org/archive/individual/xserver
 XWAYLAND_LICENSE = MIT
@@ -23,13 +23,20 @@ XWAYLAND_DEPENDENCIES = \
 	xlib_libxshmfence \
 	xlib_xtrans \
 	xorgproto
+
 XWAYLAND_CONF_OPTS = \
 	-Ddri3=true \
-	-Dxwayland_eglstream=false \
 	-Dxvfb=false \
 	-Ddefault_font_path=/usr/share/fonts/X11/ \
 	-Ddtrace=false \
 	-Ddocs=false
+# reglinux add mesa3d or img-mesa3d (PowerVR)
+ifeq ($(BR2_PACKAGE_MESA3D),y)
+XWAYLAND_DEPENDENCIES += mesa3d
+endif
+ifeq ($(BR2_PACKAGE_IMG_MESA3D),y)
+XWAYLAND_DEPENDENCIES += img-mesa3d
+endif
 
 ifeq ($(BR2_PACKAGE_LIBEPOXY),y)
 XWAYLAND_CONF_OPTS += -Dglamor=true
